Makefile

```diff
@@ -28,10 +42,15 @@ test-unit:
 test-integration:
 	RUST_LOG=$(LOG_LEVEL) cargo test --test integration --no-fail-fast -- --skip spinup_tests --skip cloud_tests --nocapture
 
-.PHONY: test-e2e
-test-e2e:
-	RUST_LOG=$(LOG_LEVEL) cargo test --test integration --features e2e-tests --no-fail-fast  -- integration_tests::test_dependencies --skip spinup_tests --skip cloud_tests --nocapture
-	RUST_LOG=$(LOG_LEVEL) cargo test --test integration --features e2e-tests --no-fail-fast -- --skip integration_tests::test_dependencies --skip spinup_tests --skip cloud_tests --nocapture
+.PHONY: test-fermyon-platform
+test-fermyon-platform:
+	RUST_LOG=$(LOG_LEVEL) cargo test --test integration --features fermyon-platform --no-fail-fast -- integration_tests::test_dependencies --skip spinup_tests --nocapture
+	RUST_LOG=$(LOG_LEVEL) cargo test --test integration --features fermyon-platform --no-fail-fast -- --skip integration_tests::test_dependencies --skip spinup_tests --nocapture
+
+.PHONY: test-spin-up
+test-spin-up:
```

Comment:
Can the [e2e step](https://github.com/fermyon/spin/blob/main/.github/workflows/build.yml#L157-L175) in the GH action also use this make target?  Would be great if this was the canonical set of commands invoked to run the tests, used by automation and devs, etc.

Reply:
yes that sounds good. I've made changes for that now.